cdc/sink/mysql/mysql_worker.go

```diff
@@ -81,20 +81,6 @@ func (w *mysqlSinkWorker) run(ctx context.Context) (err error) {
 		txnNum     int
 	)
 
-	// mark FinishWg before worker exits, all data txns can be omitted.
```

Comment:
why is this removed?

Reply:
Should we call cleanup here instead of calling it externally？

Reply:
`worker.cleanup` do the same job here:https://github.com/pingcap/tiflow/pull/5400/files#diff-ba27cb35a4f83fcd923c1403ff1fb7c1ad29659e7eadd87dd505700bf82f92b9R446 

So this defer func here seems redundant.

Reply:
I have moved the `worker.cleanup` into `worker.run`, please help me review again. @amyangfei @CharlesCheung96